samples/database.sample.yml

```diff
@@ -0,0 +1,670 @@
+# Mappings to SQLLite Database
+#
+# FST lemma             = lemma.cw
+# FST stem              = fstStem
+# homograph key         = key
+# head                  = lemma.sro
+# inflectional category = senses > sense.category
+
+- fstStem:          kinêpiko-maskotêw
+  generalWordClass: N
```

Comment:
The entry is an inflected form of _kinêpiko-maskotêw_, but in Cree that form is treated as a particle (hence the category `INM`) which may not be inflected further.